Billing.API/Models/InvoiceListItemLink.cs

```diff
@@ -0,0 +1,9 @@
+namespace Billing.API.Models
+{
+    public class InvoiceListItemLink
```

Comment:
What do you think about naming these simply _Links_? all links have the same structure.
```suggestion
    public class Link
```

Reply:
yes, for me is a good change, I put the same comment above.